pkg/block/fetcher.go

```diff
@@ -530,7 +530,7 @@ func NewLabelShardedMetaFilter(relabelConfig []*relabel.Config) *LabelShardedMet
 	return &LabelShardedMetaFilter{relabelConfig: relabelConfig}
 }
 
-// Special label that will have an ULID of the meta.json being referenced to.
+// BlockIDLabel Special label that will have an ULID of the meta.json being referenced to.
 const BlockIDLabel = "__block_id"
```

Comment:
This change is unnecessary. The original comment is better.

Reply:
Restored.